suds/wsdl.py

```diff
@@ -664,7 +664,7 @@ def body(self, definitions, body, root):
         if parts is None:
             body.parts = ()
         else:
-            body.parts = re.split("[\s,]", parts)
+            body.parts = re.split("[\\s,]", parts)
```

Comment:
or perhaps we should mark the string raw, e.g. `r"..."`

Reply:
Tomato-tomato, potato-potato. I can change if you insist :smile: